### PR TITLE
Fixes unable to run when installed project specific

### LINKED ISSUE #4468: ErrorException thrown when attempting to run from project specific
For configuration or updating questions please read the README and UPGRADE documentation,
or visit: https://gitter.im/PHP-CS-Fixer

When reporting an issue (bug) please provide the following information:

#### The PHP version you are using (`$ php -v`):
=> 7.3.7

#### PHP CS Fixer version you are using (`$ php-cs-fixer -V`):
=> 2.15.1

#### The command you use to run PHP CS Fixer:
=> `php <path to project>/_vendor/bin/php-cs-fixer -h`

When installed as project specific instead of globally this exception will be thrown:
```
PHP Warning:  Uncaught ErrorException: require_once(<path to project>/_vendor/bin/vendor/autoload.php): failed to open stream: No such file or directory in <path to project>/_vendor/bin/php-cs-fixer:61
Stack trace:
#0 <path to project>/_vendor/bin/php-cs-fixer(61): {closure}(2, 'require_once(<p...', '<path to projec...', 61, Array)
#1 <path to project>/_vendor/bin/php-cs-fixer(61): require_once()
#2 {main}
  thrown in <path to project>/_vendor/bin/php-cs-fixer on line 61
PHP Stack trace:
PHP   1. {main}() <path to project>/_vendor/bin/php-cs-fixer:0
PHP Fatal error:  main(): Failed opening required '<path to project>/_vendor/bin/vendor/autoload.php' (include_path='.:/usr/share/php') in <path to project>/_vendor/bin/php-cs-fixer on line 61
PHP Stack trace:
PHP   1. {main}() <path to project>/_vendor/bin/php-cs-fixer:0

### PR DESCRIPTION
**Pull request by ticktackk**
Fixes #4468

### CODE REVIEW

File: php-cs-fixer [hunk 1]
```diff
@@ -57,11 +57,13 @@ if ($require) {
     // OK, it's not, let give Composer autoloader a try!
     if (file_exists($a = __DIR__.'/../../autoload.php')) {
         require_once $a;
+    } elseif (file_exists($b = __DIR__.'/../autoload.php')) {
```
Comment (kubawerlos) [inline php-cs-fixer]: Can we change to `foreach` like in [here](https://github.com/sebastianbergmann/phpunit/blob/8.2.5/phpunit#L30)?

**Comment by SpacePossum** [comment]
please throw an exception if no file to include has been found so the app doesn't just crash with a more obscure error

**Comment by kubawerlos** [comment]
@SpacePossum yeah, something like [this](https://github.com/sebastianbergmann/phpunit/blob/8.2.5/phpunit#L40), right?

**Comment by SpacePossum** [comment]
@ see https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4490

**Pull request closed without merging**